assets/wire/actions.js

```diff
@@ -20,17 +20,30 @@ export function setActive(item) {
 }
 
 export const PREVIEW_ITEM = 'PREVIEW_ITEM';
-export function previewItem(item) {
+export function preview(item) {
     return {type: PREVIEW_ITEM, item};
 }
 
+export function previewItem(item) {
+    return (dispatch, getState) => {
+        if (item) {
+            markItemRead(item._id, getMaxVersion(getState().readItems[item._id], item.version));
```

Comment:
imo would be better to have `markItemRead(item, getState())` and handle this version check in utils, otherwise it's duplicated. other option would be to add a function here which would be used in both previewItem and openItem